tsconfig.json

```diff
@@ -4,11 +4,12 @@
     "removeComments": false,
     "preserveConstEnums": true,
     "sourceMap": true,
-    "declaration": true,
+    // "declaration": true,
```

Comment:
not allowed with `allowJs`, going to back that part out (and leave the lower half)

I'll bring this back in another PR later.